sdk-extensions/autoconfigure/src/test/java/io/opentelemetry/sdk/autoconfigure/LogRecordExporterConfigurationTest.java

```diff
@@ -58,21 +55,8 @@ void configureLogRecordExporters_multipleWithNone() {
                 LogRecordExporterConfiguration.configureLogRecordExporters(
                     config,
                     LogRecordExporterConfiguration.class.getClassLoader(),
-                    MeterProvider.noop(),
                     (a, unused) -> a))
         .isInstanceOf(ConfigurationException.class)
         .hasMessageContaining("otel.logs.exporter contains none along with other exporters");
   }
-
-  @Test
-  void configureOtlpLogs_unsupportedProtocol() {
```

Comment:
why is this test no longer necessary?

Reply:
Just an oversight. Wasn't possible to run in the base test module anymore since the logic has moved to the otlp module which isn't included in the base test module. I've restored it in `testFullConfig` under `LogRecordExporterConfigurationTest`. 👍